databuilder/models/watermark.py

```diff
@@ -5,20 +5,21 @@
     RELATION_END_LABEL, RELATION_TYPE, RELATION_REVERSE_TYPE
 
 
-class HiveWatermark(Neo4jCsvSerializable):
+class Watermark(Neo4jCsvSerializable):
```

Comment:
I would prefer to keep the mode file as it is if possible given the ticket is to add sample data. If we decide to support watermark for other databases other than hive, we could change the model file as well.

Reply:
@feng-tao, I've used approach proposed by @jornh so now we have both HiveWatermark (with deprecation warning) and Watermark classes. Hope this will be enough for backward compatibility.

To my mind multiple databases support for watermarks is an important feature (with only HiveWatermark I can't even specify partitions for DynamoDB table from the sample data :) ) but if you consider these changes as irrelevant for this PR - I can extract them into a separate follow-up PR.

Reply:
I think the current one is good, thanks for making it backward compatible:)